src/client/js/view/help.js

```diff
@@ -278,6 +278,7 @@ class HelpView extends BaseView {
 
     const arrowWidth = 20;
     const distanceToArrow = 35;
+    $welcome.width(640); // to force fit text lines in welcome body
```

Comment:
Should we set this to a percentage?

Reply:
I'm not sure that is necessary. Most of the other elements have a fixed size as well.

Reply:
It would be cool to set the width after finding the longest line of text in the body and then converting that to a pixel amount somehow. I don't have the jquery/html/js mojo/juju/skillz to do that, so went for the quick fix!

Reply:
I've played with stuff like that before, but there's no guarantee a user's browser/system has the same font, though I'm not sure if that applies to fonts used in javascript... But yeah, I agree: cool.

Reply:
I don't think we need to worry about how the lines break, just that the text matches what Dr. Leifer provided.

I did take a look at the site on my phone, and the menus and legend etc. seem to use a responsive design, but the welcome pane does not. I don't think it's a big deal, but my guess is that there are some classes/widgets from bootstrap, etc. that the app uses that we could take advantage of.